app/controllers/images_controller.rb

```diff
@@ -45,49 +50,51 @@ def edit
   end
 
   def update
-    result = Images::Update.call(params: params, user: current_user)
+    result = Images::UpdateInteractor.call(params: params, user: current_user)
+
+    edition, image_revision, issues, lead_selected, lead_removed =
```

Comment:
Non-blocking, but wouldn't it be clearer (and one less line of code) to assign these variables individually? e.g. `edition = result[:edition]`
I'm just not clear on what value converting result to a hash and using `value_at` is giving us.

Reply:
Yeah I think you're right in this case. 

The intention with the values_at is to do something of a mass assign at the start to bring all the variables into scope to avoid temptation of falling back to `result.*` everywhere.

In most cases we shouldn't have too many of them that this becomes a problem, but here it's a bit grim. I'll change this up.